cmd/uplink/cmd/cp.go

```diff
@@ -88,6 +90,17 @@ func upload(ctx context.Context, bs buckets.Store, srcFile string, destObj *url.
 		return err
 	}
 
+	/* create a signal of type os.Signal */
+	c := make(chan os.Signal, 0x01)
```

Comment:
Using hex numbers where we usually use decimals is against clear code and takes more effort for reading and understanding it.

Reply:
will change to decimal.